portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,23 +31,55 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
     public ArrayList<String> list = new ArrayList<String>();
+    ArrayList<String> comments = new ArrayList<String>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    list.add("Abi");
-    list.add("Patchaiyappan");
-    list.add("19");
-    String json = "{";
-    json += "\"FirstName\": ";
-    json += "\"" + list.get(0) + "\"";
-    json += ", ";
-    json += "\"LastName\": ";
-    json += "\"" + list.get(1) + "\"";
-    json += ", ";
-    json += "\"Age\": ";
-    json += "\"" + list.get(2) + "\"";
-    json += "}";
-    response.setContentType("application/json");
-    response.getWriter().println(json);
+    //Query
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("Task");
+
+    PreparedQuery results = datastore.prepare(query);
+
+    List<String> tasks = new ArrayList<>();
+    for (Entity entity : results.asIterable()) {
+      long id = entity.getKey().getId();
+      String comment = (String) entity.getProperty("comment");
+    
+      String task = comment;
+      tasks.add(task);
+    }
+
+    Gson gson = new GsonBuilder().serializeNulls().create();
+    response.setContentType("application/json;");
+    response.getWriter().println(gson.toJson(tasks));
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = getParameter(request, "comment-input", "");
+
+    // adds to stored list of comments
+    comments.add(text);
+
```

Comment:
You can delete this, since there's no need for the servlet to store comments anymore since you're transitioning to DataStore
```suggestion
```